NewEmergmed/sackett/widgets.py

```diff
@@ -215,4 +213,20 @@ def render_option(self, selected_choices, option_value, option_label, option_dat
                            data_html,
                            force_str(option_label))
 
+    def render(self, name, value, attrs=None, choices=(), renderer=None):
+        if value is None:
+            value = ''
+        final_attrs = {**self.attrs, **attrs, **{'name': name}}
+        control_id = final_attrs['id']
+
+        output = [format_html('<select name="{}" id="{}" value = "{}">',
+                                name,
+                              control_id,
+                              value), 
+                  format_html('<option value="">choose</option>')]
```

Comment:
This needs to be localised so that the current language is used to translate the word "choose" - something like this should work:

`format_html('<option value="">{}</option>',  _('choose'))`